lib/etcdv3/auth.rb

```diff
@@ -1,44 +1,46 @@
 
 class Etcdv3
   class Auth
+    attr_reader :timeout
```

Comment:
Now that timeout is being passed into the initializer, you can probably ditch this.